ArduCopter/Copter.h

```diff
@@ -501,7 +501,7 @@ class Copter : public AP_HAL::HAL::Callbacks {
 
     // Rally library
 #if AC_RALLY == ENABLED
-    AP_Rally_Copter rally{ahrs};
+    AP_Rally_Copter rally;
```

Comment:
We need to make sure this is actually calling the constructor, otherwise the `_last_change_time_ms` initialization isn't likely happening.

Reply:
I tested, and yes it got called.

I also added a simple test for rally points in Rover.